### PR TITLE
Update henry.py

### PR DESCRIPTION
**Pull request by wanda-carlson**
Test

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Testing

File: src/henry.py [hunk 1]
```diff
@@ -21,4 +21,7 @@
 print("Henry testing v0.8.0")
 print("Henry testing v0.8.0")
 print("Henry testing v0.8.0")
+print("Henry testing v0.8.0")
```
Comment (henry-richard) [inline src/henry.py:24]: Test

**Review by henry-richard — CHANGES_REQUESTED** [review]

File: src/henry.py (hunk 1 above)
Comment (henry-richard) [inline src/henry.py:24]: Test

File: src/henry.py [hunk 2]
```diff
@@ -21,4 +21,7 @@
...
+print("Henry testing v0.8.0")
```
Comment (henry-richard) [inline src/henry.py:25]: Test

**Review by henry-richard — APPROVED** [review]

**Review by henry-richard — APPROVED** [review]

**Review by henry-richard — APPROVED** [review]

**Pull request merged**